.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -47,7 +47,11 @@ install_packaged_qemu() {
 }
 
 build_and_install_qemu() {
-        PACKAGING_REPO="github.com/kata-containers/packaging"
+	if [ -n "$(command -v qemu-system-${QEMU_ARCH})" ] && [ -n "$KATA_DEV_MODE" ]; then
+		die "Qemu will not be installed"
```

Comment:
Hi~
The Stable version 2.11 isn't applied for AArch64. our qemu, for now, needed extra patch, so building from scratch is the only way. ;). I have detailed this fact [here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#build-a-custom-qemu-for-aarch64arm64---required) in doc.
So FWIT, maybe please remove [ -n "$KATA_DEV_MODE" ]. ;)

Reply:
Thanks @Pennyzct for replying

`[ -n "$KATA_DEV_MODE" ]` was added to avoid overwriting the version  of QEMU installed by the developer

cc @marcov @jodh-intel @grahamwhaley

Reply:
Indeed, `KATA_DEV_MODE` should never be set in the CI, and only set by a developer hand-running scripts locally in a manner they understand. @Pennyzct , is there a specific reason you'd like that check dropped?